tools/integration_tests/operations/operations_test.go

```diff
@@ -87,7 +87,8 @@ func TestMain(m *testing.M) {
 	setup.ParseSetUpFlags()
 
 	flags := [][]string{{"--implicit-dirs=true"},
-		{"--implicit-dirs=false"}}
+		{"--implicit-dirs=false"},
```

Comment:
please add the command for this flag combination to https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/run_tests_mounted_directory.sh